Example/SBTUITestTunnel_Tests/MiscellaneousTests.swift

```diff
@@ -240,4 +240,22 @@ class MiscellaneousTests: XCTestCase {
         
         XCTAssertFalse(app.scrollViews["scrollView"].buttons["Button"].isHittable)
     }
+
+    func testLaunchWithoutURL() {
```

Comment:
good idea. I updated the test and realized there's a [known bug](https://developer.apple.com/forums/thread/25355?answerId=755101022#755101022) with XCUIApplication.openURL where both the url isn't present in the launch options for application:didFinishLaunching and application:openURL is not called.

would you prefer to merge this so it's available once this bug is fixed, or hold off on making this change for now?

Reply:
The PR is basically good to go, but regrettably the feature still doesn't work on the latest beta. So, I think it's better to hold off on making this change since I'm unsure whether this will be fixed anytime soon. If we proceed now the library will wrap a malfunctioning feature which could mislead users into thinking there's a problem with the library itself, rather than it being an issue with XCUITest (which is honestly hard to believe).

On a related note to these changes, at Subito we're using https://github.com/Subito-it/SBTUITestTunnelHost which allows to interact with `simctl` enabling us to test cold/hot deeplinks by invoking `simctl open`. While it might not be as elegant as using the intended API, it could serve as an alternative solution.

Any thoughts?

Reply:
that makes sense to me! I'll close this out and we can revisit if / when Apple gets around to fixing the bug. I'll look into the host library you shared. @tcamin thanks for the quick feedback!